lib/hexpm/web/views/layout_view.ex

```diff
@@ -42,9 +42,17 @@ defmodule Hexpm.Web.LayoutView do
       tag(:meta, property: "og:title", content: Map.get(assigns, :title)),
       tag(:meta, property: "og:type", content: "website"),
       tag(:meta, property: "og:url", content: Map.get(assigns, :canonical_url)),
-      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/hex.png")),
+      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/open-graph.png")),
+      tag(:meta, property: "og:image:width", content: "1200"),
+      tag(:meta, property: "og:image:height", content: "630"),
       tag(:meta, property: "og:description", content: description(assigns)),
-      tag(:meta, property: "og:site_name", content: "Hex")
+      tag(:meta, property: "og:site_name", content: "Hex"),
+      tag(:meta, property: "twitter:card", content: "summary_large_image"),
+      tag(:meta, property: "twitter:site", content: "@hex_pm"),
+      tag(:meta, property: "twitter:creator", content: "@hex_pm"),
```

Comment:
Good point. I'd say let's implement as little as we need for hex.pm links to not look broken on social media :) Perhaps we can just stick to `og:*` links? Seems this is what GH does:

```html
<meta content="https://avatars1.githubusercontent.com/u/6621265?v=3&amp;s=400" property="og:image" /><meta content="GitHub" property="og:site_name" /><meta content="object" property="og:type" /><meta content="hexpm/hexpm" property="og:title" /><meta content="https://github.com/hexpm/hexpm" property="og:url" /><meta content="hexpm - API server and website for Hex" property="og:description" />
```

<view-source:https://github.com/hexpm/hexpm>